scripts/entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "${RUNNER_NAME}" ]; then
+  echo "RUNNER_NAME must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_TOKEN}" ]; then
+  echo "RUNNER_TOKEN must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_ORG}" ]; then
+  echo "RUNNER_ORG must be set" 1>&2
+  exit 1
+fi
+
+if [ -z "${RUNNER_REPO}" ]; then
+  echo "RUNNER_REPO must be set" 1>&2
+  exit 1
+fi
+
+cd /runner
+mkdir -p _work
+./config.sh --unattended --replace --name "${RUNNER_NAME}" --url "https://github.com/${RUNNER_ORG}/${RUNNER_REPO}" --token "${RUNNER_TOKEN}" --work /runner/_work
+
+# TODO: run placemat
+
+./bin/runsvc.sh
```

Comment:
I want to know what this script is.
Does it do long polling and run actions?

Reply:
`runsvc.sh` is a script officially provided by GitHub to run long polling and downloading assigned job scripts basically.

`runsvc.sh` invokes [RunnerService.js](https://github.com/actions/runner/blob/8109c962f09d9acc473d92c595ff43afceddb347/src/Misc/layoutbin/runsvc.sh#L16) and it invokes [Runner.Listener](https://github.com/actions/runner/blob/8109c962f09d9acc473d92c595ff43afceddb347/src/Misc/layoutbin/RunnerService.js#L20) developed in C#.
So, precisely speaking, `Runner.Listerner` is responsible for the long polling and script downloading, and `runsvc.sh` and `RunnerService.js` are responsible for handling error and restarting the long polling process to keep the process alive as long as possible.